package.json

```diff
@@ -116,15 +115,11 @@
     "expect": "^1.20.2",
     "exports-loader": "^0.6.3",
     "file-loader": "^1.1.11",
-    "gulp": "^3.9.1",
-    "gulp-nodemon": "^2.2.1",
-    "gulp-util": "^3.0.7",
     "ignore-styles": "^5.0.1",
     "imports-loader": "0.7.0",
     "json-loader": "^0.5.4",
     "mini-css-extract-plugin": "0.4.0",
     "mocha": "3.2.0",
-    "ngrok": "2.2.4",
     "nock": "^9.0.2",
     "nodemon": "^1.11.0",
```

Comment:
I guess we can remove `nodemon` and `open` too since they are no longer being used anywhere.

Reply:
You are absolutely right. I'll remove them